src/colorize.cr

```diff
@@ -145,36 +145,40 @@ module Colorize
     LightCyan    = 96
     White        = 97
 
-    def fore
-      to_i.to_s
+    def fore(io : IO) : Nil
+      to_i.to_s io
     end
 
-    def back
-      (to_i + 10).to_s
+    def back(io : IO) : Nil
+      (to_i + 10).to_s io
     end
   end
 
   record Color256,
     value : UInt8 do
-    def fore
-      "38;5;#{value}"
+    def fore(io : IO) : Nil
+      io << "38;5;"
+      value.to_s io
     end
 
-    def back
-      "48;5;#{value}"
+    def back(io : IO) : Nil
+      io << "48;5;"
+      value.to_s io
     end
   end
 
   record ColorRGB,
     red : UInt8,
     green : UInt8,
     blue : UInt8 do
-    def fore
-      "38;2;#{red};#{green};#{blue}"
+    def fore(io : IO) : Nil
+      io << "38;2;"
+      {red, green, blue}.join(";", io, &.to_s io)
```

Comment:
Some nitpicking:
- Maybe use a Char `';'` instead of a String `";"`.
- Maybe not the most readable solution but that's OK.